Makefile

```diff
@@ -533,38 +533,38 @@ test-wasm: $(WASM_MODULE_OUTPUTS)  ### Run engine tests
 
 .PHONY: bench
 bench: ## Run benchmarks in /benches
-	${MAYBE_ENVIRONMENT_EXEC} cargo bench --no-default-features --features "benches" ${CARGO_BENCH_FLAGS}
+	${MAYBE_ENVIRONMENT_EXEC} cargo bench --no-default-features --features "benches" ${CARGO_BENCH_FLAGS:-""}
```

Comment:
A question just for my education, what does the addition here do? I'm not super informed about how makefiles work beyond a basic grasp.

Reply:
The end result here is that we're avoiding passing an empty string argument by having it do some "if `$CARGO_BENCH_FLAGS` is empty, substitute it with `""`", which as I say it, yes, I realize this sounds silly... but I guess the way Make does variable substitution... it was passing the empty variable but then when I slapped some default value substitution logic, the shell word trimming logic kicked in and then culled it entirely, leading to the functional outcome of "if it's empty, don't pass it at all".

Practically speaking, this is actually fixed in #7574 because of @jszwedko's keen eye that we can just not set a default value at the export callsite on line 21.